api/trip/models.py

```diff
@@ -32,7 +32,7 @@ class Trip(models.Model):
     activities = models.ManyToManyField('trip.Activity')
     start_date = models.DateField()
     end_date = models.DateField()
-    attendees = models.ManyToManyField('accounts.CustomUser')
+    # attendees = models.ManyToManyField('accounts.CustomUser')
```

Comment:
Is this line of code needed at all? If not, could you just remove it all together rather than commenting it out?

Reply:
It doesn't need. Sorry about this. I already removed it and fixed the conflicts.